torch/_export/converter.py

```diff
@@ -380,7 +380,7 @@ def convert_graph_outputs(self):
                 )
             )
 
-        self.fx_graph.output(args)
+        self.fx_graph.output(args[0])  # Get rid of an extra list wrapped around final output.
```

Comment:
I would double check this assumption on models with multiple outputs.

e.g. `return a, b`, is different from `return (a, b)`

Reply:
I added a test case to verify that, but I think from TorchScript point of view, they are all treated as a TupleConstruct. 

Even thinking from the native python function, those two variables will anyway be packed in a single tuple [ref](https://stackoverflow.com/questions/38508/whats-the-best-way-to-return-multiple-values-from-a-function) when the function returns.